dashboard/app/models/level.rb

```diff
@@ -69,7 +69,8 @@ def serializable_hash(options=nil)
   # Rails won't natively assign one-to-one association attributes for
   # us, even though we've specified accepts_nested_attributes_for above.
   # So, we must do it manually.
-  def assign_attributes(attributes)
+  def assign_attributes(new_attributes)
+    attributes = new_attributes.stringify_keys
```

Comment:
It's simply [mirroring some functionality in `super`](https://github.com/rails/rails/blob/8b26431ba4cec4a5401dd1e5e50c15c11bec0703/activerecord/lib/active_record/attribute_assignment.rb#L29) to ensure that the next line will work the same whether this method was called with `'level_concept_difficulty'` or `:level_concept_difficulty` as an attribute.